From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Sun, 30 Oct 2016 14:42:45 -0400
Subject: Avoid warning during key import

Without this patch, after importing keys, i see this on the
thunderbird error console:

    Warning: Expected ':' but found ';'.  Declaration dropped.
    Source File: chrome://enigmail/content/enigmailKeyImportInfo.xul
    Line: 0, Column: 5
    Source Code:
    plain;white-space: pre
    -----^
---
 ui/content/enigmailKeyImportInfo.xul | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/content/enigmailKeyImportInfo.xul b/ui/content/enigmailKeyImportInfo.xul
index 5a46b3b..6732631 100644
--- a/ui/content/enigmailKeyImportInfo.xul
+++ b/ui/content/enigmailKeyImportInfo.xul
@@ -31,7 +31,7 @@
 
   <vbox align="center" flex="1" style="overflow:auto" id="outerbox">
     <hbox align="center" flex="1">
-      <description flex="1" id="keyInfo" style="plain;white-space: pre"/>
+      <description flex="1" id="keyInfo" class="plain" style="white-space: pre"/>
     </hbox>
   </vbox>
 
